.github/workflows/ci.yml

```diff
@@ -0,0 +1,29 @@
+name: CI
+
+on: push
```

Comment:
Looks like you didn't update you script before you ran this.
We need pull_request here, too, and on the correct branch.

Reply:
Yeesh. I could have sworn I updated it. I’ll wrap through and fix that.